core/app/models/spree/refund.rb

```diff
@@ -33,22 +30,23 @@ def description
       payment.payment_method.name
     end
 
-    private
-
     # attempts to perform the refund.
     # raises an error if the refund fails.
     def perform!
       return true if transaction_id.present?
 
       credit_cents = Spree::Money.new(amount.to_f, currency: payment.currency).money.cents
 
-      @response = process!(credit_cents)
+      response = process!(credit_cents)
+      log_entries.create!(details: response.to_yaml)
 
-      self.transaction_id = @response.authorization
+      self.transaction_id = response.authorization
```

Comment:
Thank you for the feedback, Jordan! Forgive me, I can only reply to comments on the code.

You are correct that this change would potentially break `Refund.total_amount_reimbursed_for`,
so I have added a scope to make sure the method only performs on transacted refunds. 

The goal of this PR is to remove the implicit call of `perform!` so that refunds could be created
without automatically performing them, for the purposes of data importing